libvast/vast/system/actors.hpp

```diff
@@ -14,11 +14,35 @@
 
 #include <caf/replies_to.hpp>
 
+#include <filesystem>
+
 #define VAST_ADD_TYPE_ID(type) CAF_ADD_TYPE_ID(vast_actors, type)
 
+// NOLINTNEXTLINE(cert-dcl58-cpp)
+namespace std::filesystem {
```

Comment:
Quite ugly that `inspect` needs to be in `std::filesystem` namespace. AFAIK this is in violation of the standard saying this is UB, but if caf requires it...shame on them.

Reply:
Lame, but I accept that given that it's treated as a 3rd party lib. When can we upgrade to CAF 0.18? I know you've expressed interest before in it.

Reply:
The migration is a really large amount of work. If nothing urgent comes up, I plan to start working on it sometime this month, but will only spend one day per week on it at most. We will not, however, drop CAF 0.17.6 support for a while, so there'll be a ifdefs for a few months.